packages/paste-website/src/utils/stickyUtils.ts

```diff
@@ -1,15 +1,5 @@
-import {PSA_ALERT_HEIGHT, PASTE_THEME_ALERT_HEIGHT, SITE_MASTHEAD_HEIGHT} from '../constants';
+import {PSA_ALERT_HEIGHT, SITE_MASTHEAD_HEIGHT} from '../constants';
 
-export const getMainContentComputedOffset = (isPasteTheme: boolean): number => {
-  if (isPasteTheme) {
-    return PSA_ALERT_HEIGHT + PASTE_THEME_ALERT_HEIGHT;
-  }
-  return PSA_ALERT_HEIGHT;
-};
-
-export const getStickyColumnComputedOffset = (isPasteTheme: boolean, topPad: number = 20): number => {
-  if (isPasteTheme) {
-    return topPad + PASTE_THEME_ALERT_HEIGHT + PSA_ALERT_HEIGHT + SITE_MASTHEAD_HEIGHT;
-  }
-  return topPad + PSA_ALERT_HEIGHT + SITE_MASTHEAD_HEIGHT;
-};
+const topPad = 20;
+export const getStickyColumnComputedOffset = topPad + PSA_ALERT_HEIGHT + SITE_MASTHEAD_HEIGHT;
+export const getStickySidebarComputedOffset = PSA_ALERT_HEIGHT + SITE_MASTHEAD_HEIGHT;
```

Comment:
Stick all these in `constants`. Change the name to something like `STICKY_PAGE_COLUMN_OFFSET` `STICKY_SIDEBAR_OFFSET`, get rid of `topPad`, just hard code it in the constant